tests/integration/mounts_propagation.bats

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+        requires root
+        setup_debian
+}
+
+function teardown() {
+        teardown_bundle
+}
+
+@test "runc run [rootfsPropagation shared]" {
+        # Set rootfsPropagation to shared
```

Comment:
This is obvious from the code and does not need to be explained in a code comment line